Lib/test/test_opcache.py

```diff
@@ -1256,6 +1256,37 @@ def g():
         self.assert_no_opcode(g, "CONTAINS_OP")
 
 
+    @cpython_only
+    @requires_specialization_ft
+    def test_unpack_sequence(self):
+        def f():
+            for _ in range(100):
+                a, b = 1, 2
```

Comment:
Sort of unrelated to this PR, but I'm surprised constant unpacking like this isn't peepholed into `LOAD_CONST, STORE_FAST, LOAD_CONST, STORE_FAST`. Maybe because it's actually a longer instruction sequence than the original? It certainly does less work.

@iritkatriel, thoughts?